### PR TITLE
Documentation

### LINKED ISSUE #28: Improve readme
Improve readme documentation making sure it reflect latest 1.0

### PR DESCRIPTION
**Pull request by nickbalestra**
- [x] I ran `npm test` for the package I'm modifying
- [x] I used `npm run commit` instead of `git commit`

As per #28 ,
Closes #28, #26, #17

### CODE REVIEW

**Comment by nickbalestra** [comment]
@geovanisouza92 Wait for merging this. 
I'm going to push to this PR commit related to #26 and #17

**Comment by geovanisouza92** [comment]
@nickbalestra Is it ready? I'm asking because there is a [WIP] on title...

**Comment by geovanisouza92** [comment]
So, will this PR close #17, #26 and #28?

**Comment by nickbalestra** [comment]
@geovanisouza92 done, ready to be merged.
Yes, this close #17, #26 and #28.

**Comment by geovanisouza92** [comment]
Awesome! This is a way better than before. The images are so beautiful that I have to lick the screen :joy: 

About the images, it's possible to move them to the repo too?

**Comment by nickbalestra** [comment]
Yes. good idea. Let me do it

**Comment by nickbalestra** [comment]
@geovanisouza92 should be done and ready to go now, over to you.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+4. Run `npm install`.
+5. Write code.
+6. Run `npm test` to lint and test. Don’t commit before fixing all errors and warnings.
+7. Commit using `npm run commit` and follow the CLI instructions.
+8. Make a pull request.
+
+# To release new versions
+
+1. Check that you have npm publishing rights before anything else.
+2. Run `npm run check-release`.
```
Comment (geovanisouza92) [inline CONTRIBUTING.md]: There is such script?

File: README.md [hunk 2]
```diff
@@ -1,95 +1,190 @@
...
+Inside that directory, it will generate the following initial project structure* and install the required dependencies.
 
-- [cycle-scripts-es-browserify](./cycle-scripts-es-browserify/): ES6 (babel) + Browserify
-- [cycle-scripts-ts-browserify](./cycle-scripts-ts-browserify/): TypeScript + Browserify
-- [cycle-scripts-es-webpack](./cycle-scripts-es-webpack/): ES6 (babel) + Webpack
-- [cycle-scripts-ts-webpack](./cycle-scripts-ts-webpack/): TypeScript + Webpack
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
```
Comment (geovanisouza92) [inline README.md]: `puplic` typo

File: README.md [hunk 3]
```diff
@@ -1,95 +1,190 @@
...
+_* Generated structure and files may change depending on the flavor being used, The above structure hold true for any of the 4 core flavors._
+
+No configuration or complicated folder structures, just the files you need to build your cycle app.
+Once the installation is done, you can run the following commands from within the project folder:
+
+### `npm start`
+
+Runs the app in development mode by starting the development server.
+
+Core flavors come with [ live | hot module ] reload so that the page will reload if you make edits.
```
Comment (geovanisouza92) [inline README.md]: Maybe we should omit this part about "hot module reload/replacement", as we didn't ship this, yet.

File: README.md [hunk 4]
```diff
@@ -1,95 +1,190 @@
...
 
-## No lock-in
+Copy dependencies and configurations to the project folder, update package.json and remove the dependency on the flavored cycle-scripts.
 
-This is a tool focused on Cycle.js beginners and to provide fast bootstrap for new projects, and doesn't have the ambition to be **the** tool for working with Cycle.js projects in the long term. With that in mind, it's easy to leave `create-cycle-app` defaults and follow your own steps, by running `npm run take-off-training-wheels`.
+**This is irreversible.**
 
-# Custom flavor
+## Flavors
+Each flavor represents a pair of programming language and builds tool. All the underlying dependencies and configuration are hidden behind the flavor. All flavors must adhere to the same basic structure and commands. Please make sure to check a specific flavor documentation for more details.
```
Comment (geovanisouza92) [inline README.md]: Is "builds tool" a typo here?

File: cycle-scripts-es-browserify/README.md [hunk 5]
```diff
@@ -1,4 +1,39 @@
...
+
+
+## Boilerplate:
+
+The flavor generate the following file structure:
+
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
```
Comment (geovanisouza92) [inline cycle-scripts-es-browserify/README.md]: `puplic` typo

File: cycle-scripts-es-webpack/README.md [hunk 6]
```diff
@@ -1,4 +1,44 @@
...
+
+
+## Boilerplate:
+
+The flavor generate the following file structure:
+
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
```
Comment (geovanisouza92) [inline cycle-scripts-es-webpack/README.md]: `puplic` typo

File: cycle-scripts-ts-browserify/README.md [hunk 7]
```diff
@@ -1,4 +1,42 @@
...
+
+
+## Boilerplate:
+
+The flavor generate the following file structure:
+
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
```
Comment (geovanisouza92) [inline cycle-scripts-ts-browserify/README.md]: `puplic` typo

File: cycle-scripts-ts-webpack/README.md [hunk 8]
```diff
@@ -1,4 +1,44 @@
...
+
+
+## Boilerplate:
+
+The flavor generate the following file structure:
+
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
```
Comment (geovanisouza92) [inline cycle-scripts-ts-webpack/README.md]: `puplic` typo

File: README.md (hunk 4 above)
Comment (nickbalestra) [inline README.md]: Dunno, grammarly suggested me so...

File: CONTRIBUTING.md (hunk 1 above)
Comment (nickbalestra) [inline CONTRIBUTING.md]: There is now, i saw we were using on other cycle repos, nice little northbrook feature!

**Comment by nickbalestra** [comment]
@geovanisouza92 :Wow, you better the grammarly! 
Good catches and some good points, updated the above.

Thansk!

**Comment by geovanisouza92** [comment]
Merged :tada:

**Pull request merged**